packages/ci/src/utils/test-run.ts

```diff
@@ -127,16 +129,20 @@ function draw(testRuns: Heroku.TestRun[], watchOption = false, jsonOption = fals
     )
   })
 
-  cli.table(data, {
-    printHeader: undefined,
-    columns: [
-      {key: 'iconStatus', width: 1, label: ''}, // label '' is to make sure that widh is 1 character
-      {key: 'number', label: ''},
-      {key: 'branch'},
-      {key: 'sha'},
-      {key: 'status'},
-    ],
-  })
```

Comment:
Any suggestions on how to verify this change? The interface appears to have an undocumented breaking change, and this at least calms TS. It doesn’t appear to have any direct test coverage and I don’t know the intended outcome.

Reply:
It looks like it should be called whenever the ci command is run? So it should run if you test the package locally I would think. I just tried it and errors stopped me - could be from the TS upgrade or something I was doing.

Reply:
I ran yarn test from the packages/ci directory.

Reply:
It seems to be working for me.